### ISSUE TITLE
The result of MOT17 with private detection

### ISSUE THREAD

**Issue by TTTREE** [xingyizhou/CenterTrack#200]
Hi!
Thanks for your great work!
I am confusing about the experiment setting on the MOT17 private detection version, which MOTA=67.3.
1. Do you use CrowdHuman pretraining on this private pipeline?  
   Because I get a lower result than 67.3 only training on the MOT17 train set and cannot find information about it in your paper.
   I read the document in the project carefully and find "mot17_fulltrain are finetuned on the crowdhuman model".
   I think this means the private detection of MOTA67.3 is pretraining on crowdhuman, am I right?

2. And then, what is performance on MOT17 test set with private detection only training on MOT17 training set?
    Have you tested it?

Could you help me, please?
Thanks!!

**Comment by xingyizhou** [comment]
Hi,
  Thank you for your interest in our work.

  1. Yes you are right. we always use crowdhuman pretraining for our private model. We should have explicitly stated this in the paper. 
  2. We didn't evaluate any additional results other than the numbers in the paper on the test set. In our supplementary, we performed ablation experiments of using crowdhuman on the half-half train/ val split.

Best,
Xingyi
